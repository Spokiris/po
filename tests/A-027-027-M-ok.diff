*** tests/A-027-027-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-027-027-M-ok.outhyp	2023-10-27 19:48:41.112640512 +0100
***************
*** 9,16 ****
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Valor a procurar: 1;1|15
! 1;3|15=1;1
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
--- 9,19 ----
  1 - Procurar valores
  2 - Procurar funções
  0 - Sair
! Escolha uma opção: Menu de Consultas
! 1 - Procurar valores
! 2 - Procurar funções
! 0 - Sair
! Escolha uma opção: Opção inválida!
  Menu de Consultas
  1 - Procurar valores
  2 - Procurar funções
