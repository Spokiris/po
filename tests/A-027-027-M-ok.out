Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 5
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: 1
Valor a procurar: 15
1;1|15
1;3|15=1;1
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 0
